packages/pytest/build

```diff
@@ -3,6 +3,4 @@ source /opt/mesosphere/environment.export
 export LIB_INSTALL_DIR="$PKG_PATH/lib/python3.5/site-packages"
 mkdir -p "$LIB_INSTALL_DIR"
 
-for package in pytest teamcity-messages; do
-  pip3 install --no-deps --install-option="--prefix=$PKG_PATH" --root=/ /pkg/src/$package
-done
+pip3 install --no-deps --install-option="--prefix=$PKG_PATH" --root=/ /pkg/src/pytest
```

Comment:
If moving to a single python package build, please copy / paste the standard template (the pytest in this line should be $PKG_NAME): 

https://github.com/dcos/dcos/blob/master/packages/python-docopt/build#L6